docs/wiki/deployment/process-auditing.md

```diff
@@ -8,28 +12,93 @@ to your query schedule, or to a query pack.
 
 ## Linux process auditing
 
-osquery can use Linux Audit on the supported Linux distributions. This does NOT require any audit configuration or `auditd`; actually, auditd should not be running if using osquery's process auditing.
+osquery uses the Linux Audit System to collect and process audit events from the kernel. It accomplishes this by monitoring the `execve()` syscall. Auditd should not be running when using osquery's process auditing, as it will conflict with osqueryd over access to the audit netlink socket. You should also ensure auditd is not configured to start at boot.
 
-This creates a bit of confusion since audit, auditd, and libaudit are ambiguous-- osquery only uses the audit features in the kernel. Most distributions do not install libaudit or auditd, this is perfectly fine. If you are configuring audit, using a control binary, or `/etc/audit.conf`, your osquery *may* override your settings.
+The only prerequisite for using osquery's auditing functionality on Linux is that you must use a kernel version that contains the audit functionality. Most kernels over version 2.6 have this capability.
```

Comment:
I moved it to line 48 which seemed more relevant about osquery overriding settings since that flag is actually what controls whether or not your settings will be overwritten - happy to move it back if you think that makes more sense